src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -96,6 +97,29 @@ public function iProceedSelectingOfflinePaymentMethod($paymentMethodName)
     }
 
     /**
+     * @When /^I proceed selecting "([^"]*)" shipping method$/
```

Comment:
@pamil Nope, because this step select shipping method by name (from form) so object is not needed here.